Use the Calypso theme standard colors for the domain statuses

hambai opened the pull request:
#### Changes proposed in this Pull Request

* Instead of hardcoding the status colors we can use the standard theme colors for warning/error/success

Here's how it looks:

<img width="746" alt="Screenshot 2020-07-22 at 10 10 15" src="https://user-images.githubusercontent.com/1355045/88145894-d0e72980-cc03-11ea-9286-3dd7e2fb4c38.png">
<img width="743" alt="Screenshot 2020-07-22 at 10 10 32" src="https://user-images.githubusercontent.com/1355045/88145897-d2b0ed00-cc03-11ea-9a78-7dc8dfbba44b.png">
<img width="744" alt="Screenshot 2020-07-22 at 10 11 23" src="https://user-images.githubusercontent.com/1355045/88145902-d3498380-cc03-11ea-9290-85cf0ff9de4a.png">
<img width="747" alt="Screenshot 2020-07-22 at 10 11 45" src="https://user-images.githubusercontent.com/1355045/88145905-d3e21a00-cc03-11ea-8652-2f79b4720251.png">


#### Testing instructions

* Check domains in different statuses - expired, expiring soon and activating and verify that the colors look okay

fditrapani commented:
Thanks for this change @hambai! Can we make the activating green instead of grey? That way it looks like more of a positive thing. Other than that this looks great!

hambai commented:
@fditrapani okay, so I updated the Activating status to use the success color. I also updated the Parked status to use the color scheme neutral color which is slightly different than what you gave us in the mockups - hope that's okay?

fditrapani commented:
> I also updated the Parked status to use the color scheme neutral color which is slightly different than what you gave us in the mockups - hope that's okay?

Looks great! Thanks for the update @hambai!
